OJP/OJP_Availability.xsd

```diff
@@ -0,0 +1,426 @@
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="bicycle">
+				<xs:annotation>
+					<xs:documentation>Space for a bicycle.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="guideDog">
+				<xs:annotation>
+					<xs:documentation>Place suitable when accompanied by a guide dog.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="dog">
+				<xs:annotation>
+					<xs:documentation>Place suitable when accompanied by a dog.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="boardingAssistance">
+				<xs:annotation>
+					<xs:documentation>Assistance for boarding.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="alightingAssistance">
+				<xs:annotation>
+					<xs:documentation>Assistance for unboarding.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="onboardAssistance">
+				<xs:annotation>
+					<xs:documentation>Assistance available on board, during the journey.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="unassistedMinorAssistance">
+				<xs:annotation>
+					<xs:documentation>Assistance for a minor travelling alone.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:annotation>
+		<xs:documentation>========================================== Problems ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:simpleType name="AvailabilityProblemTypeEnumeration">
+		<xs:annotation>
+			<xs:documentation>Types of problems that may be returned in responses to STOPEVENT requests.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="AVAILIBILITY_NOSERIVCEFOUND">
+				<xs:annotation>
+					<xs:documentation>No SERVICE (VEHICLE JOURNEY or VEHICLE) could be found for this request.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="AVAILABILITY_LOCATIONUNSERVED">
+				<xs:annotation>
+					<xs:documentation>At the locations (address, stop, etc.) for which the availability has been requested, there is no public transport service at all. E.g. outside of the area</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="AVAILABILITY_LOCATIONUNKNOWN">
+				<xs:annotation>
+					<xs:documentation>The location (address, stop, etc.) for which availability was requested is unknown.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="AVAILABILITY_OUTSIDERULES">
+				<xs:annotation>
+					<xs:documentation>The service is not availability by one of the rules (e.g. outside operating hours, shadowing an existing public transport. In the details the service should inform, which rule was violated by the request.</xs:documentation>
```

Comment:
Sentence should be improved & clarified. "The service violates one or several rules..." (which / what kind of rules is not clear to me; neither do I understand what "shadowing" means)